includes/libraries/parsecsv.lib.php

```diff
@@ -0,0 +1,1163 @@
+<?php
+
+class parseCSV {
```

Comment:
The class exists check should be done in this class as opposed to where it is currently done (comment https://github.com/easydigitaldownloads/easy-digital-downloads/pull/4609/files/e121a4b13f965285ca566e12b5f1e34632b49b11#r64472008). That way any extensions can also include the file without having to worry about things.